Scripts tested on Ubuntu & New Ansible playbooks

GONZALORUIZ opened the pull request:
Scripts have been tested on Ubuntu and I have added an ansible playbook that will create a RAID device on all the VMs with all the available data disks

GONZALORUIZ committed:
Ubuntu 12 and CentOS 6.6 tested

control_path added in ansible.cfg file
Removed unnecessary comments

GONZALORUIZ committed:
Ansible playbook for configuring storage added

The Ansible playbook will create a RAID device in all the VMs

singhkays commented on ansible-advancedlinux/metadata.json:
```diff
@@ -1,7 +1,7 @@
-  "description": "Manage your  Linux VMs efficiently using Ansible.This advanced template deploys N number of Linux VMs and it configures Ansible and SSH",
-  "summary": "This advanced template deploys N number of Linux VMs and it configures Ansible so you can easily manage all the VMS. Don't suffer more pain configuring and managing all your VMs , just use Ansible! Ansible is a very powerful masterless configuration management system based on SSH.Do you have 1 or 10 or 100 VMs? It won't make any difference with Ansible",
+  "description": "Manage your  Linux VMs efficiently using Ansible.This advanced template deploys N number of Linux VMs and it configures Ansible and SSH.Then using an Ansible playbook it will create a RAID with all the available data disks",
+  "summary": "This advanced template deploys N number of Linux VMs and it configures Ansible so you can easily manage all the VMS.Additionally,it will execute an ansible Playbook to create a RAID with all the available data disks. Don't suffer more pain configuring and managing all your VMs , just use Ansible! Ansible is a very powerful masterless configuration management system based on SSH.Do y ...
```
@GONZALORUIZ Can you reduce this to less than 200 characters? You can use description for the long text up to 1000 characters.

GONZALORUIZ committed:
Removed --list-hosts and spaces in the playbook

GONZALORUIZ committed:
Indentation fixed

Indentation fixed for the Ansible playbook
Reduce the tracing level for ansible

GONZALORUIZ committed:
Convention rules applied to metadata.json

Summary field reduced to 199 characters.

GONZALORUIZ commented:
Thanks Kay, I have incorporated your suggestions.

GONZALORUIZ committed:
Support for xfs filesystem tested on ubuntu

xfsprogs installation added on the playbook
